x-pack/solutions/observability/plugins/synthetics/server/tasks/sync_private_locations_monitors_task.test.ts

```diff
@@ -288,6 +311,7 @@ describe('SyncPrivateLocationMonitorsTask', () => {
       const res = await task.hasAnyDataChanged({
         taskState: taskState as any,
         soClient: mockSoClient as any,
+        lastStartedAt: expect.anything(),
```

Comment:
Should we pass a date string here? Not sure why you're using `expect.anything()`.